modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -804,17 +807,16 @@ private CompletableFuture<Void> startPartitionAndStartClient(
 
         var raftNodeId = localMemberAssignment == null ? null : new RaftNodeId(replicaGrpId, serverPeer);
 
-        boolean shouldStartRaftListeners = localMemberAssignment != null && !((Loza) raftMgr).isStarted(raftNodeId);
+        boolean shouldStartPartitionTrackers = localMemberAssignment != null && !((Loza) raftMgr).isStarted(raftNodeId);
```

Comment:
Who is partition tracker?

Reply:
I realised that the better naming is `shouldUpdatePartitionTrackers`, will update. I decided to rename it because originally the name `shouldStartRaftListeners` had no meaning  for me because I had the same question about "Raft Listeners"

Reply:
PartitionListerner aka RaftListener is a common one, PartitionTracker is not.

Reply:
reverted